Readme.md

```diff
@@ -246,7 +244,20 @@ println aecu.contentUpgradeBuilder()
         .run()
 ```
 
-<a name="jmx"></a>
+### Filter by Node Path regex
```

Comment:
Filter by Node Path regex -> Filter by Node Path
Fits better to other headlines and we might add more methods.